playground/backend/pkg/fs_tool/java_fs.go

```diff
@@ -0,0 +1,111 @@
+package fs_tool
+
+import (
+	"fmt"
+	"io/fs"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+)
+
+const (
+	executableFilesFolderName = "executable_files"
+	srcFolder                 = "src"
+	binFolder                 = "bin"
+)
+
+type JavaFileSystemService struct{}
+
+func (jFS *JavaFileSystemService) PrepareFolders() error {
+	// Create executable_files/src folder
+	err := os.MkdirAll(filepath.Join(executableFilesFolderName, srcFolder), fs.ModePerm)
```

Comment:
it will create folder in backend, not in pkg

Reply:
Changed